docs/_user_guide/best_practices_screen.md

```diff
@@ -2,102 +2,62 @@
 
-You may run into a situation where you are disconnected or timed out from the Yens while you are attached to a screen session. When you log back onto the Yens and try to reconnect to that session again, you may find that you cannot re-attach to the screen, since it's already "Attached". In order to return to that session, you can just use this command:
+To set up a screen, `ssh` in to the Yens through a terminal window and then type:
+```title="Terminal Command"
+screen -S <session_name>
 ```
-screen -d -r <sessionname>
-```
-
-This detaches the previous connection to that screen session and then returns you to the screen.
-
-Finally, once you are finished with a screen session, all you have to do is type:
-```
-exit
-```
-
-## Screen
-Generally speaking, you can use the `screen` command, described in more detail below, to continue having your scripts run on the 
-interactive Yens after logging out. 
+where `<session_name>` is any name of your choice.
 
+!!! Warning
+	Do not initiate a screen in a terminal on JupyterHub. The screen will time out and you will not be able to access it later. Please only initiate screens through a SSH connection to the Yens outside of JupyterHub.
 
-<a href="https://www.gnu.org/software/screen/manual/screen.html" target="_blank">`screen`</a> is a window manager that allows you to set up 
-multiple virtual sessions within your terminal session. This can be useful for running multiple processes at the same time, 
-but it can also be used to leave processes running after you log out of any interactive Yen server.
-
-To set up a screen, use:
-
-```bash
-$ screen -S <sessionname>
-```
+!!! Note 
+	Screens are not tied to your user, but rather to the specific Yen that you started the screen in. So do not expect your screen session on `yen1` to appear in `yen2`.
 
 For example, let's name our session `npv`:
 
-```bash
+```title="Terminal Command"
 $ screen -S npv
 ```
 
 This will take you to a new terminal with a blank view and a command prompt at the top of the page. 
 Here, you can start issuing commands as usual and in particular, you can start a script that you expect to run for a long time.
 
-{% include note.html content="Screens are not tied to your user, but rather to the specific Yen that you started the screen in. So do not expect your screen session on `yen1` to appear in `yen2`." %}
-
 Let's run our serial R script again. This time, we will let it run in a screen (if you had modules loaded, 
 starting a screen preserves these in the new terminal started by `screen`). So, we can start our R process:
 
-```bash
-$ Rscript investment-npv-parallel.R 
+```title="Terminal Command"
+$ Rscript investment-npv-parallel.R
 ```
 
 Once your script is going, you can exit out of this screen session with the keyboard shortcut `ctrl + a + d`, which will return you to your original terminal. 
```

Comment:
Can we do keyboard formatting like they do here?

https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#keyboard-shortcuts-docsjavascriptsshortcutsjs
https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/docs/setup/setting-up-navigation.md

Reply:
I don't know how to do this. Can someone please help?

Reply:
Thank you Jeff for the help!